artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireConverter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.core.protocol.openwire;
+
+import org.apache.activemq.artemis.api.core.ICoreMessage;
+import org.apache.activemq.artemis.core.message.impl.CoreMessageObjectPools;
+import org.apache.activemq.artemis.spi.core.protocol.MessageConverter;
+
+public class OpenWireConverter implements MessageConverter<OpenwireMessage> {
+
+   private static final OpenWireConverter theInstance = new OpenWireConverter();
+   private OpenWireConverter() {
+   }
+
+   public static OpenWireConverter getInstance() {
+      return theInstance;
+   }
+
+   @Override
+   public OpenwireMessage fromCore(ICoreMessage coreMessage) throws Exception {
```

Comment:
seems one is left, i think the class was there before unused, would you mind? OpenwireMessage -> OpenWireMessage

Reply:
Ok, @michaelandrepearce I will refractor that. Since this was named like this previously I thought not to change it. As after what you suggested this was bugging me too.